dnf-plugins-core.spec

```diff
@@ -415,6 +415,29 @@ Download module metadata from all enabled repositories, module artifacts and pro
 repository.
 %endif
 
+%if %{with python3}
+%package -n python3-dnf-plugin-system-upgrade
+Summary:        System Upgrade Plugin for DNF
+Requires:       python3-systemd
+Provides:       dnf-command(system-upgrade)
+Provides:       dnf-command(offline-upgrade)
+Provides:       dnf-command(offline-distrosync)
+Provides:       dnf-plugin-system-upgrade = %{version}-%{release}
+Provides:       python3-dnf-plugins-extras-system-upgrade = %{version}-%{release}
+Obsoletes:      fedup < 0.9.4
+Obsoletes:      dnf-plugin-system-upgrade < 0.10
+Obsoletes:      python3-dnf-plugins-extras-system-upgrade < %{dnf_plugins_extra}
+Conflicts:      python2-dnf-plugin-system-upgrade < %{version}-%{release}
+BuildRequires:  pkgconfig(systemd)
+BuildRequires:  systemd
+BuildRequires:  python3-systemd
+%{?systemd_requires}
+
+%description -n python3-dnf-plugin-system-upgrade
+System Upgrade Plugin for DNF. Enables offline system upgrades and distrosync
+using three commands: ``system-upgrade``, ``offline-upgrade``, and ``offline-distrosync``.
+%endif
```

Comment:
Is there a reason we should have this split out rather than part of the main dnf-plugins-core package like the rest of the plugins?

Reply:
Hello, I was following the latest plugins added to core like `modulesync` that are also located in separate packages. Are there any advantages now having it in the main package? I thought the main package is there mainly because different approach was used back then and for backward compatibility of the original commands from core.

Reply:
We will have a discussion regarding the moving into the core package across the teams and I'll let you know.